crates/bevy_ui/src/render/mod.rs

```diff
@@ -155,6 +163,9 @@ pub struct ExtractedUiNode {
     // it is defaulted to a single camera if only one exists.
     // Nodes with ambiguous camera will be ignored.
     pub camera_entity: Entity,
+    pub border_radius: [f32; 4],
```

Comment:
Doc strings on these fields: the order of the floats is important and non-obvious.

Reply:
What about:
```rust
/// Border radius of the UI node.
/// Ordering: top left, top right, bottom right, bottom left.
pub border_radius: [f32; 4],
```

Reply:
That's exactly what I was hoping for.